tools/arxiv/scripts/collect_ai_papers.py

```
@@ -0,0 +1,358 @@
+- Large Language Models (LLMs)
+- Actor Network Theory (ANT)
+
+Targets: 5000+ papers for comprehensive testing
+"""
+
+import json
+import time
+import logging
+from pathlib import Path
+from datetime import datetime, timedelta
+from typing import List, Dict, Set
```

Ruff (0.12.2)

`typing.List` is deprecated, use `list` instead

(UP035)